source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Migrate to Drupal 9 on Pantheon
+subtitle: Migrate a Drupal 9 Site to Pantheon
+description: Migrate a Composer-managed Drupal 9 Site from another platform to Pantheon.
+categories: [develop]
+cms: drupal-9
+tags: [code, launch, migrate, site, updates]
+reviewed: "2021-08-20"
+layout: guide
+showtoc: true
+permalink: docs/guides/drupal-9-migration/drupal-9-to-pantheon
+anchorid: drupal-9-migration/drupal-9-to-pantheon
+editpath: drupal-9-migration/05-drupal-9-to-pantheon.md
+---
+
+In this doc, you'll migrate an existing Composer-managed Drupal 9 site from another platform to a new Drupal 9 site with Integrated Composer on Pantheon.
+
+## Unresolved Questions / TO-DOs
+
+- [How] Do we address nested docroot vs not-nested docroot? - **Probably not necessary**
```

Comment:
address the subject of whether the existing site had a nested docroot and what it was named.

[original comment](https://github.com/pantheon-systems/documentation/pull/6682#discussion_r715065396)